src/coreclr/jit/gentree.cpp

```diff
@@ -21017,6 +21017,22 @@ GenTree* Compiler::gtNewSimdBinOpNode(
 
             return gtNewSimdBinOpNode(GT_AND, type, shiftOp, maskOp, simdBaseJitType, simdSize);
         }
+
+        case GT_LSH:
+        {
+            assert(varTypeIsByte(simdBaseType));
+            unsigned shiftCountMask = (genTypeSize(simdBaseType) * 8) - 1;
+            GenTree* andShiftVal =
+                gtNewOperNode(GT_AND, op2ForLookup->TypeGet(), op2ForLookup, gtNewIconNode(shiftCountMask));
+            GenTree* op3 =
+                gtNewSimdBinOpNode(GT_LSH, type, op1, andShiftVal,
+                                   varTypeIsUnsigned(simdBaseType) ? CORINFO_TYPE_USHORT : CORINFO_TYPE_SHORT,
+                                   simdSize);
+            GenTree* maskElement =
+                gtNewOperNode(GT_LSH, andShiftVal->TypeGet(), gtNewIconNode(0xFF), gtCloneExpr(andShiftVal));
```

Comment:
We want to use `fgMakeMultiUse` not `gtCloneExpr`, much like is done in the `GT_RSZ` handling.